tailwind.config.js

```diff
@@ -112,10 +112,15 @@ module.exports = {
           '0%': { opacity: '0.6' },
           '50%': { opacity: '1' },
           '100%': { opacity: '0.6' }
-        }
+        },
+        slideIn: {
```

Comment:
want to get @CITIZENDOT and @aditya-mitra 's opinion on these tailwind config changes

Reply:
no issues with this config, can safely merge.